zerver/openapi/zulip.yaml

```diff
@@ -12936,6 +12950,19 @@ paths:
                     type: integer
                   default: []
                   example: [1, 10]
+                user_group_ids:
+                  description: |
+                    A list containing the [IDs of the user groups](/api/get-user-groups) that
+                    the newly created user will be automatically added to if the invitation
+                    is accepted. If the list is empty, then the new user will not be
+                    added to any user groups.
+
+                    **Changes**: New in Zulip 9.0 (feature level 2\_\_).
```

Comment:
The feature level here is corrupted. Looks like you did some sort of bugged regex substitution; generally in my view it's better to renumber by hand; it's only a few values to change, and you avoid bugs like this.